src/platforms/kotlin/index.mdx

```diff
@@ -0,0 +1,8 @@
+Sentry supports JVM related use of Kotlin with our [Java](/platforms/java) SDK, and, for Android applications, with our [Android](/platforms/android) platform.
+
+Select your platform to get started:
+
+- [Java](/platforms/java)
+- [Android](/platforms/android)
+
+We are also working on an [experimental SDK for Kotlin Multiplatform](https://github.com/getsentry/sentry-kotlin-multiplatform). Feedback welcome.
```

Comment:
this was just a hack week project, maybe this would create a false expectation?

Reply:
I think we'd like to keep the link to see if folks upvote it.